src/Events/CacheResolveEvent.php

```diff
@@ -45,7 +45,7 @@ class CacheResolveEvent extends BCEvent
     /**
      * Resource url.
      *
-     * @var null
+     * @var string|null
```

Comment:
i think we should add the type declaration to the property, then we can remove the annotation completely.